src/xdocs/beginning_development.xml

```diff
@@ -49,6 +49,12 @@
         <a href="idea.html">IntelliJ IDEA IDE</a><br/>
       </p>
 
+      <p>
+        Video walk through of these instructions is available in this
+        <a href="https://www.youtube.com/playlist?list=PLHM9s_lN4X0hzOQ0sUmGdroxW0HfREAqj">
+           playlist</a>
```

Comment:
`playlist` => `YouTube playlist`

"playlist on YouTube" is also acceptable.

Reply:
Done